flink-table/flink-table-common/src/main/java/org/apache/flink/table/factories/CatalogStoreFactory.java

```diff
@@ -108,7 +108,7 @@
 public interface CatalogStoreFactory extends Factory {
 
     /** Creates a {@link CatalogStore} instance from context information. */
-    CatalogStore createCatalogStore(Context context);
+    CatalogStore createCatalogStore();
```

Comment:
This change should be updated to the FLIP?

Reply:
I think yes, do you think this change is necessary? I think the revision may be more reasonable